v03_pipeline/lib/reference_datasets/mitomap.py

```diff
@@ -0,0 +1,29 @@
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+# adapted from download_and_create_reference_datasets/v02/mito/write_mito_mitomap_ht.py
+def get_ht(raw_dataset_path: str, reference_genome: ReferenceGenome) -> hl.Table:
+    ht = hl.import_table(
+        raw_dataset_path,
+        delimiter=',',
+        types={'"Position"': hl.tstr, '"Allele"': hl.tstr},
+    )
+    # NB: all strings in the downloaded CSV file are wrapped in double quotes
```

Comment:
import_table takes a `quote` arguments - does using that resolve this issue?

Reply:
yes! thanks